files.cpp

```diff
@@ -32,7 +32,6 @@ namespace files {
 			return path;
 		}
 
-		path = path.lexically_normal();
```

Comment:
this is important

Reply:
This is duplicated, there's another `lexically_normal` call above. Unless calling it twice is required?

Reply:
Oh I see, definitely shouldn’t be required twice. Wonder how it got that way 🤔